yarn.lock

```diff
@@ -2897,7 +2911,7 @@
     yargs "^13.2.4"
     yarn-install "^1.0.0"
 
-"@wdio/config@5.13.2", "@wdio/config@^5.12.1", "@wdio/config@^5.9.4":
```

Comment:
Something weird is going on here with the selective dependency resolution in the root package.json. This is what's causing E2ETest failure. Need to take a closer look at this.